### PR TITLE
Removed static content

### PR DESCRIPTION
**Pull request by horato**
And created like bilion merge conflicts

### CODE REVIEW

**Comment by MythicManiac** [comment]
Instead of changing the platform target, this should be made compatible with 32-bit environments.

I get that the issue is with the .ependency being 64-bit only, but that should be fixed ASAP. If you want to merge these changes before that, I'd suggest extracting the test to it's own PR and writing a test for this that won't require said incompatible dependency.

**Comment by MythicManiac** [comment]
Besides what I just pointed out, this did seem alright with a quick glance

File: GameServer/Logic/Enet/ClientInfo.cs [hunk 1]
```diff
@@ -11,114 +11,101 @@ namespace LeagueSandbox.GameServer.Logic.Enet
...
-        Peer peer;
+        public bool KeyChecked = false;
+        public bool VersionMatch = true;
+        public long UserId;
+        public int SkinNo = 0;
+        public SummonerSpellIds[] SummonerSkills = new SummonerSpellIds[2];
+        public string Name, Rank;
+        public short Ribbon;
+        public int Icon;
+        public TeamId Team;
```
Comment (MythicManiac) [inline GameServer/Logic/Enet/ClientInfo.cs]: These should be properties, so
`public TeamId Team { get; protected set; }` or something similar.

File: GameServer/Logic/Config.cs [hunk 2]
```diff
@@ -10,66 +10,71 @@
...
+    public class Config
     {
-        public static Dictionary<string, PlayerConfig> players;
-        public static GameConfig gameConfig;
-        public static MapSpawns mapSpawns;
-        public static ContentManager contentManager;
-        public static string version = "Version 4.20.0.315 [PUBLIC]";
+        public Dictionary<string, PlayerConfig> Players;
+        public GameConfig GameConfig;
+        public MapSpawns MapSpawns;
```
Comment (MythicManiac) [inline GameServer/Logic/Config.cs]: Same here.

Properties or private variables

File: GameServer/Logic/Config.cs [hunk 3]
```diff
@@ -112,17 +117,17 @@ public GameConfig(JToken gameData)
...
-        public string summoner2 { get { return (string)_playerData.SelectToken("summoner2"); } }
-        public short ribbon { get { return (short)_playerData.SelectToken("ribbon"); } }
-        public int icon { get { return (int)_playerData.SelectToken("icon"); } }
+        public string Rank { get { return (string)_playerData.SelectToken("rank"); } }
+        public string Name { get { return (string)_playerData.SelectToken("name"); } }
+        public string Champion { get { return (string)_playerData.SelectToken("champion"); } }
+        public string Team { get { return (string)_playerData.SelectToken("team"); } }
+        public short Skin { get { return (short)_playerData.SelectToken("skin"); } }
+        public string Summoner1 { get { return (string)_playerData.SelectToken("summoner1"); } }
+        public string Summoner2 { get { return (string)_playerData.SelectToken("summoner2"); } }
```
Comment (MythicManiac) [inline GameServer/Logic/Config.cs]: Also here

File: GameServer/Logic/Config.cs [hunk 4]
```diff
@@ -87,21 +92,21 @@ public PlayerSpawns(JArray spawns)
...
         }
     }
 
-    internal class GameConfig
+    public class GameConfig
     {
-        public int map { get { return (int)_gameData.SelectToken("map"); } }
-        public string gameMode { get { return (string)_gameData.SelectToken("gameMode"); } }
+        public int Map { get { return (int)_gameData.SelectToken("map"); } }
+        public string GameMode { get { return (string)_gameData.SelectToken("gameMode"); } }
```
Comment (MythicManiac) [inline GameServer/Logic/Config.cs]: And here :)

File: GameServer/Logic/Game.cs [hunk 5]
```diff
@@ -136,78 +138,93 @@ private void update()
...
         {
             return _map;
         }
 
+        public PacketNotifier GetPacketNotifier()
+        {
+            return _packetNotifier;
+        }
+
+        public PacketHandlerManager GetPacketHandlerManager()
```
Comment (MythicManiac) [inline GameServer/Logic/Game.cs]: Use getter / setter properties please.

`public PacketHandlerManager PacketHandlerManager { get; set; }`

**Comment by MythicManiac** [comment]
Well now the build and tests succeed at least

File: GameServer/Logic/Config.cs (hunk 3 above)
Comment (horato) [inline GameServer/Logic/Config.cs]: here what? Its property already.

Comment (MythicManiac) [inline GameServer/Logic/Config.cs]: Oh what
Github showed invalid diff for me, so nevermind this one

File: GameServer/Logic/Config.cs (hunk 4 above)
Comment (MythicManiac) [inline GameServer/Logic/Config.cs]: Same here, github screwing up

**Comment by MythicManiac** [comment]
This increases code coverage, but codecov is drunk. Merging

**Pull request merged**